omega/omega-transaction/pom.xml

```diff
@@ -30,6 +30,10 @@
   <name>Saga::Omega::Transaction</name>
 
   <dependencies>
+    <dependency>
```

Comment:
if we really want to share code between omega and alpha, is it better to do it in a separate module?

Reply:
yeah, I agree. How about the "common" module ?

Reply:
ok, pack-common sounds good